rpcs3/rpcs3qt/instruction_editor_dialog.cpp

```diff
@@ -84,7 +84,8 @@ instruction_editor_dialog::instruction_editor_dialog(QWidget *parent, u32 _pc, c
 void instruction_editor_dialog::updatePreview()
 {
 	bool ok;
-	ulong opcode = t2_instr->text().toULong(&ok, 16);
```

Comment:
couldn't you just not assign it to a variable xD ?
that way we don't have a warning